src/tests/unit/platform/test_alertsv7_api.py

```diff
@@ -568,6 +568,121 @@ def on_post(url, body, **kwargs):
                  {"field": {}, "values": [{"id": "status", "name": "statusX", "total": 9}]}]
 
 
+def test_query_intrusion_detection_system_with_all_bells_and_whistles(cbcsdk_mock):
+    """Test a IDS alert query with all options selected."""
+
+    def on_post(url, body, **kwargs):
+        assert body == {"query": "Blort",
```

Comment:
Context for other comments - when setting up mock requests that validate the request generated by the SDK, it should be runnable in postman.